com.github.xournalpp.xournalpp.yaml

```diff
@@ -0,0 +1,104 @@
+app-id: com.github.xournalpp.xournalpp
+runtime: org.gnome.Platform
+runtime-version: '3.32'
+sdk: org.gnome.Sdk
+command: xournalpp
+rename-icon: xournalpp
+rename-desktop-file: xournalpp.desktop
```

Comment:
It would be nice if you can rename those upstream & maybe upstream the appdata file.

Reply:
sure. but it requires upstream to change their id though, no?

Reply:
It's just a suggestion, yes. Let's see what upstream think about that first.